entries/checkboxradio.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0"?>
+<entry name="checkboxradio" type="widget" widget-element="checkboxradio">
+	<title>Checkboxradio Widget</title>
+	<desc>Converts inputs of type radio and checkbox to themeable buttons.</desc>
+	<longdesc>
+		<p>Native HTML input elements are impossible to style consistently. This widget allows working around that limitation by positining the associated label on top of the hidden input, and emulating the checkbox or radio element itself using an (optional) icon. The original input still receives focus and all events, the label merely provides a themeable button on top.</p>
+
+		<xi:include href="../includes/widget-theming.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
+
+		<ul>
+			<li>
+				<code>ui-checkboxradio</code>: The input of type radio or checkbox. Will be hidden, with its associated label positioned on top.
+				<ul>
+					<li>
+						<code>ui-checkboxradio-label</code>: The label associated with the input. If the input is checked, this will also get the <code>ui-checkboxradio-checked</code> class. If the input is of type radio, this will also get the <code>ui-checkboxradio-radio-label</code> class.
+					</li>
+					<li>
+						<code>ui-checkboxradio-icon</code>: If the <a href="#option-icon"><code>icon</code></a> option is enabled, the generated icon has this class.
+					</li>
+					<li>
+						<code>ui-checkboxradio-icon-space</code>: If the <a href="#option-icon"><code>icon</code></a> option is enabled, an extra element with this class as added between the text label and the icon.
+					</li>
+				</ul>
+			</li>
+
+		</ul>
+
+		<h3>Dependencies</h3>
+		<ul>
+			<li><a href="/labels/">.labels()</a></li>
+			<li><a href="/form-reset-mixin/">Form Reset Mixin</a></li>
```

Comment:
Added form-reset-mixin. Still don't know what to do about `button.css` or `escape-selector`. I think I'd rather just remove these sections altogether and work on requirejs and webpack plugins for parsing our css comments instead.